tsl/src/remote/dist_ddl.c

```diff
@@ -390,9 +420,23 @@ dist_ddl_preprocess(ProcessUtilityArgs *args)
 			break;
 
 		case T_CreateTrigStmt:
+		{
+			CreateTrigStmt *stmt = castNode(CreateTrigStmt, args->parsetree);
+
+			if (stmt->funcname != NIL)
+			{
+				Oid funcargtypes[1];
+				Oid funcid = LookupFuncName(stmt->funcname,
+											0 /* nargs */,
```

Comment:
If arguments are unknown, LookupFuncName expects nargs to be set to -1.

Reply:
In this case we know that the function we are looking for takes 0 arguments, so passing in nargs=0 is what we want to do (trigger functions never takes any arguments). PG12 doesn't allow passing in NULL argtypes, however, as it asserts that argtypes is always non-NULL. PG13 changed that to allow NULL in case of nargs=0, but that is not compatible with PG12. I will clarify in a comment.